src/openms_gui/source/VISUAL/APPLICATIONS/GUITOOLS/TOPPAS.cpp

```diff
@@ -168,10 +171,14 @@ int main(int argc, const char** argv)
 
     a.connect(&a, SIGNAL(fileOpen(QString)), mw, SLOT(openToppasFile(QString)));
 
-    // Create the splashscreen that is displayed while the application loads
-    QSplashScreen* splash_screen = new QSplashScreen(QPixmap(":/TOPPAS_Splashscreen.png"));
+    // Create the splashscreen that is displayed while the application loads (version is drawn dynamically)
+    QPixmap qpm(":/TOPPAS_Splashscreen.png");
+    QPainter pt_ver(&qpm);
+    pt_ver.setFont(QFont("Helvetica [Cronyx]", 15, 2, true));
```

Comment:
is this font available on all systems?

Reply:
yes, it should be. If not Font-matching kicks in http://doc.qt.io/qt-5/qfont.html#fontmatching
So this is safe